gluten-ut/spark32/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -265,6 +265,7 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude("to_timestamp")
     // Legacy mode is not supported, assuming this mode is not commonly used.
     .exclude("SPARK-30668: use legacy timestamp parser in to_timestamp")
+    .exclude("function to_date")
```

Comment:
Please leave some comment to tell the reason for excluding this. And also what changes have been made in the substituted test.